azerothcore/azerothcore-wotlk
# fix(DB/Creature): Tender Strider Meat chance increase /  icon for Refurbished Steam Tank

## Changes Proposed:
-  Increase drop chance of 'Tender Strider Meat' for quest Kyle's Gone Missing! (ID: 11129) to 50% based on videos that show that this quest is completed with one or two kills of a mob - currently on AC it's a 10% droprate which is considerable for a low level entry quest
-  Remove 'Get In' Icon on Refurbished Steam Tank


## Issues Addressed:
- It was closed, but #3239 mentions the low drop chance of Kyle's Gone Missing! quest item
- Refurbished Steam Tank, an NPC with SAI in Stormwind, was an NPC implemented in WotLK pre-patch - it used to fight the Scourge during the Scourge Invasion Event, but now it simply spawns, drives around and breaks down.  You were never able to 'get in' to this vehicle, as it was just an NPC that drove around and smashed some undead baddies. ([based on Wowhead comments for this npc](https://www.wowhead.com/npc=29144/refurbished-steam-tank#comments))

## How to Test the Changes:
1. Apply SQL, clear your cache and restart the server - create an alliance character for the steam tank, a horde character for the quest
2. .tele stormwindharbor and find the Steam Tank, hover your mouse over it to ensure it doesn't show a get in icon
3. .tele mulgore, .level 10 and .quest add 11129, go kill the specified mobs to ensure the drop chance was increased


## Target Branches
Master
